### PR TITLE
As a user purchasing a physical ticket, the seller is notified to mail them after purchase, closes #33, closes #12

### LINKED ISSUE #12: As a buyer after checkout, I receive an email confirmation with basic order details and link to order detail page

### LINKED ISSUE #33: As a user purchasing a physical ticket, the seller is notified to mail them after purchase

### PR DESCRIPTION
**Pull request by KristaANelson**

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 class OrdersController < ApplicationController
   def show
     @order = Order.find(params[:id])
-    @items = @order.order_items
   end
```
Comment (KristaANelson) [inline app/controllers/orders_controller.rb:4]: You can just call @order.items so @items is not needed

File: app/controllers/orders_controller.rb [hunk 2]
```diff
@@ -11,20 +10,39 @@ def new
 
   def index
     if current_user
-      @orders = Order.where("user_id = ?", current_user.id)
+      @orders = Order.where(user_id: current_user.id)
     else
```
Comment (KristaANelson) [inline app/controllers/orders_controller.rb:14]: refactored, only need the "'s for a comparison like >=

File: app/models/order.rb [hunk 3]
```diff
@@ -12,10 +12,17 @@ class Order < ActiveRecord::Base
   scope :completed, -> { where("status = ?", "completed") }
   scope :cancelled, -> { where("status = ?", "cancelled") }
 
+  def buyer
+    User.find(user_id)
+  end
+
+  def sellers
+    order_items.map(&:seller).uniq
+  end
```
Comment (KristaANelson) [inline app/models/order.rb:21]: this way you can find out easily who is the seller user and who is the buyer user on an order

File: db/seeds.rb [hunk 4]
```diff
@@ -382,7 +382,7 @@ def generate_admins
                  password_confirmation: "password",
                  password:              "password")
     Admin.create(full_name:             "Josh Cheek",
-                 email:                 "demo+josh@jumpstartlab.com",
+                 email:                 "demo+josh@example.com",
                  password_confirmation: "password",
```
Comment (KristaANelson) [inline db/seeds.rb:386]: updating to example.com so I don't send out random emails to jumpstart labs

**Comment by bhargavisatpathy** [comment]
Looks great!!!

**Pull request merged**